daos/movies.go

```diff
@@ -0,0 +1,54 @@
+package daos
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/ashwinspg/explore-golang/constants"
+	"github.com/ashwinspg/explore-golang/dtos"
+)
+
+var (
+	ErrMovieNotFoundInDB = errors.New("Movie Information for given UUID is not found in Database")
```

Comment:
variable name can just be ErrMovieNotFound
Similarly, error message too can be Movie Information for given UUID is not found in Database

If we need to know if the movie exists in database or not, we could log such details.
Such details that are of use to us alone(developers) need not be returned as error, can just be logged.